python/dpu_utils/utils/chunkwriter.py

```diff
@@ -81,3 +89,17 @@ def close(self) -> None:
         self.__flush()
         if self.__parallel_writers > 0:
             self.__writer_executors.shutdown(wait=True)
+
+    def __get_max_existing_index(self):
+        """
+        Returns the largest file index within the current output folder.
+        """
+        file_regex = re.compile(f'{self.__file_prefix}([0-9]+){self.__file_suffix}')

```

Comment:
We're trying to keep dpu-utils py3.5 compat, so could you change to a version without string interpolation?